ion_schema_2_0/imports/isl_1_0_importing_isl_2_0.isl

```diff
@@ -0,0 +1,44 @@
+$ion_schema_2_0
+
+type::{
+  name: ordinary_struct,
+  type: struct,
+}
+
+type::{
+  // This is using features not available in Ion Schema 1.0
+  name: snake_case_annotated,
+  annotations: { element: { regex: "^[a-z0-9_]+$"} },
+}
+
+$test::{
+  description: "Ion Schema 1.0 should be able to import from an Ion Schema 2.0 document",
```

Comment:
What actually happens? Should we add some assertions?

Reply:
Sure. I'm going to refactor these tests so they can assert that the types do what we expect them to do.